packages/api/src/filestorage/file/services/googledrive/index.ts

```
@@ -1,32 +1,180 @@
 @Injectable()
 export class GoogleDriveService implements IFileService {
   constructor(
     private prisma: PrismaService,
     private logger: LoggerService,
     private cryptoService: EncryptionService,
     private registry: ServiceRegistry,
+    private ingestService: IngestDataService,
+    private bullQueueService: BullQueueService,
   ) {
     this.logger.setContext(
       FileStorageObject.file.toUpperCase() + ':' + GoogleDriveService.name,
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)